docs/how-to-guides/calibrating-your-sensors.md

```diff
@@ -6,7 +6,10 @@ Autoware expects to have multiple sensors attached to the vehicle as input to pe
 
 ## Camera calibration
 
-Navigation2 provides a [good tutorial for camera internal calibration](https://navigation.ros.org/tutorials/docs/camera_calibration.html).
+### Intrinsic Calibration
+
+- Navigation2 provides a [good tutorial for camera internal calibration](https://navigation.ros.org/tutorials/docs/camera_calibration.html).
+- Autocore provides a [light-weight tool](https://github.com/autocore-ai/calibration_tools/tree/main/camera_intrinsic_calib).
```

Comment:
@JianKangEgon Is it okay to use `Autocore`, not `AutoCore`?

Reply:
Autocore is OKay

Reply:
Which is correct? I believe it's better to be consistent.

Reply:
@kenji-miyake sorry for the late reply, last week was our national holiday. I've double checked, to be consistent, AutoCore would be better, I'll close this pr and propose a new one @Sharrrrk,  many thanks